src/utils/encryption.js

```diff
@@ -21,7 +21,20 @@ const getPrimordialHash = () => {
   return CryptoJS.SHA256(entropy).toString();
 };
 
-const sha256 = message => CryptoJS.SHA256(message).toString();
+// Returns [obfuscatedHash, nextHash]
+const hashChain = msg => {
```

Comment:
Is this obfuscating the "message" field?  We will only need to obfuscate the sha256 hashes that define the addresses.  

In fact if we obfuscate the message field I don't think we'll be able to download the file, since you can't decrypt a hash.  

Disregard this comment if I am reading this wrong.

Reply:
The param name is misleading. It's the hash that its obfuscating, not the msg. I'll change it.